core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1620,7 +1632,7 @@ class SnappySession(_sc: SparkContext) extends SparkSession(_sc) {
     */
 
     sessionCatalog.registerPolicy(policyName, tableName, policyFor, applyTo, expandedPolicyApplyTo,
-      currentUser, filterStr, filter)
+      owner, filterStr, filter)
```

Comment:
Aren't owner and currentUser exactly same? The callbacks.checkSchemaPermission method returns the same as its argument as per the changes in this PR.

Reply:
Not if the owner is a ldap group.